clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -1101,6 +1101,56 @@ def BitfieldInfoAttr : CIR_Attr<"BitfieldInfo", "bitfield_info"> {
   ];
 }
 
+//===----------------------------------------------------------------------===//
+// AnnotationAttr
+//===----------------------------------------------------------------------===//
+def AnnotationAttr : CIR_Attr<"Annotation", "annotation"> {
+  let summary = "Annotation attribute for global variables and functions";
+  let description = [{
+    Represent C/C++ attribute of annotate in CIR. example c code is:
+    ```
+     in test.c line 10
```

Comment:
`in test.c line 10` -> `// in test.c line 10`

Reply:
Not sure I understand how `line 10` connects to the rest of the explanation here.